flutter/lib/src/sentry_flutter.dart

```diff
@@ -45,27 +44,26 @@ mixin SentryFlutter {
     final native = SentryNative();
     native.setNativeChannel(nativeChannel);
 
+    final wrapper = PlatformDispatcherWrapper(PlatformDispatcher.instance);
```

Comment:
The PlatformDispatcher should be accessed via one of the WidgetsBindings:

> Consider avoiding static references to this singleton through [PlatformDispatcher.instance](https://api.flutter.dev/flutter/dart-ui/PlatformDispatcher/instance.html) and instead prefer using a binding for dependency resolution such as WidgetsBinding.instance.platformDispatcher. See [PlatformDispatcher.instance](https://api.flutter.dev/flutter/dart-ui/PlatformDispatcher/instance.html) for more information about why this is preferred.

https://api.flutter.dev/flutter/dart-ui/PlatformDispatcher-class.html

Reply:
You can use the `BindingUtils.getWidgetsBindingInstance().platformDispatcher`.

Reply:
Is it ok to access the widgets binding instance at this place?

Reply:
It could be that this only works after `WidgetsFlutterBinding.ensureInitialized()` which is executed likely after? has to be tested.

Reply:
Maybe the solution to do it automatically is then to use `PlatformDispatcher.instance` despite it being discouraged since it doesn't require the WidgetsBinding to be initialized :|

Reply:
@marandaneto WDHT? Change back to `PlatformDispatcher.instance`?

Reply:
@denrase did you test this? https://github.com/getsentry/sentry-dart/pull/1039#discussion_r985701810
It was a hunch and a question, maybe it works?

Reply:
The docs says:

> The only place that WidgetsBinding.instance.platformDispatcher is inappropriate is if access to these APIs is required before the binding is initialized by invoking runApp() or WidgetsFlutterBinding.instance.ensureInitialized(). In that case, it is necessary (though unfortunate) to use the [PlatformDispatcher.instance](https://api.flutter.dev/flutter/dart-ui/PlatformDispatcher/instance.html) object statically.

So in this case, it'd be fine to use `PlatformDispatcher.instance`, in case it's necessary.

Reply:
Tested this and we indeed have a null value for `BindingUtils.getWidgetsBindingInstance()`. The approach with `PlatformDispatcher.instance works`:

<img width="811" alt="Bildschirmfoto 2022-10-10 um 11 07 24" src="https://user-images.githubusercontent.com/3984453/194832917-ea889bd2-8c3a-4238-ba09-15a2247e49da.png">

<img width="1114" alt="Bildschirmfoto 2022-10-10 um 11 08 03" src="https://user-images.githubusercontent.com/3984453/194832928-a2197dcd-9ed8-4bec-acd1-ade73a81a59a.png">